Sources/BraveShared/BraveStrings.swift

```diff
@@ -4299,6 +4291,50 @@ extension Strings {
   }
 }
 
+extension Strings {
+  public struct RecentlyClosed {
+    public static let viewRecentlyClosedTab =
+      NSLocalizedString(
+        "recently.closed.view.tab",
+        tableName: "BraveShared",
+        bundle: .module,
+        value: "View Recently Closed Tab",
```

Comment:
`s/Tab/Tabs`

Reply:
Thanks for pointing it.   Fix at https://github.com/brave/brave-ios/pull/6755/commits/85f9c7a3153b05a4134b5bb6fd15c42038813472